CHANGELOG.md

```diff
@@ -2,6 +2,18 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 
 ### [Unreleased](https://github.com/cucumber/cucumber-js/compare/v6.0.5...master) (In Git)
 
+#### Breaking changes
+
+* Drop support for Node.js 8
+* Formatters
```

Comment:
Think we also need to mention the TypeScript rewrite, as it will require users to drop their dependency on `@types/cucumber` and change a few interface names.

Reply:
Wasn't certain how that effected users (I'm new to typescript libraries)

Reply:
I'm happy to write that one up in a separate PR, I'll add to the TypeScript tasks issue